ignite/pkg/cosmosgen/generate.go

```diff
@@ -13,17 +15,29 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cosmosanalysis/module"
 	"github.com/ignite/cli/ignite/pkg/cosmosver"
 	"github.com/ignite/cli/ignite/pkg/gomodule"
+	"github.com/ignite/cli/ignite/pkg/xfilepath"
 )
 
 const (
-	moduleCacheNamespace = "generate.setup.module"
+	moduleCacheNamespace       = "generate.setup.module"
+	includeProtoCacheNamespace = "generator.includes.proto"
+)
+
+var protocGlobalInclude = xfilepath.List(
+	xfilepath.JoinFromHome(xfilepath.Path("local/include")),
+	xfilepath.JoinFromHome(xfilepath.Path(".local/include")),
```

Comment:
I wonder if we need to bring back the global include, do we need it?

Reply:
probably not.

Not quite sure what it was for....I added it as part of reverting the original code.